tests/unittests/Foundation/Makefile

```
@@ -0,0 +1,96 @@
+COPY_FILES := $(addprefix $(OUTDIR)/, $(RF_RESOURCE_FILES))
+COPY_FILES += $(OUTDIR)/data/NSFileManagerUT.txt
+
+
+INC := $(SRCDIR)/../../frameworks/include $(SRCDIR)/../../frameworks/gtest $(SRCDIR)/../../frameworks/gtest/include 
+INC += $(SRCDIR)/../../frameworks/OSXShims/include $(SRCDIR)/ReferenceFoundation $(SRCDIR)/../../../include/xplat
+INC_PARAMS := $(INC:%=-I %)
+
+CFLAGS := -std=c++14
+CFLAGS += -x objective-c++
+CFLAGS += $(INC_PARAMS)
+CFLAGS += -DTARGET_OS_MAC
+CFLAGS += -DWINOBJCRT_IMPEXP=
+CFLAGS += -mmacosx-version-min=$(OSX_VERSION_MIN)
+
+LDFLAGS := -framework Foundation -mmacosx-version-min=$(OSX_VERSION_MIN)
+
+SHIMS := $(OUT_OBJ_DIR)/windows.o $(OUT_OBJ_DIR)/IwMalloc.o $(OUT_OBJ_DIR)/LoggingNative.o
+
+$(OUT_OBJ_DIR)/IwMalloc.o : $(SRCDIR)/../../../Frameworks/WinObjCRT/MemoryManagement.cpp
+	mkdir -p $(OUTDIR)/Objects
+	$(CLANG) -c -o $@ $(CFLAGS) $<
+
+$(OUT_OBJ_DIR)/windows.o : $(SRCDIR)/../../frameworks/OSXShims/src/windows.cpp
+	mkdir -p $(OUTDIR)/Objects
+	$(CLANG) -c -o $@ $(CFLAGS) $<
+
+$(OUT_OBJ_DIR)/LoggingNative.o : $(SRCDIR)/../../frameworks/OSXShims/src/LoggingNative.mm
+	mkdir -p $(OUTDIR)/Objects
+	$(CLANG) -c -o $@ $(CFLAGS) $<
+
+$(OUT_OBJ_DIR)/%.o : $(SRCDIR)/%.m
+	mkdir -p $(OUTDIR)/Objects
+	$(CLANG) -c -o $@ $(CFLAGS) $^
+
+$(OUT_OBJ_DIR)/%.o : $(SRCDIR)/%.mm
+	mkdir -p $(OUTDIR)/Objects
+	$(CLANG) -c -o $@ $(CFLAGS) $^
+
+$(OUT_OBJ_DIR)/ReferenceFoundation_TestUtils.o : $(SRCDIR)/ReferenceFoundation/TestUtils.mm
+	mkdir -p $(OUTDIR)/Objects
+	$(CLANG) -c -o $@ $(CFLAGS) $<
+
+$(OUT_OBJ_DIR)/ReferenceFoundation_%.o : $(OUT_OBJ_DIR)/ReferenceFoundation_TestUtils.o $(SRCDIR)/ReferenceFoundation/%.mm
+	mkdir -p $(OUTDIR)
+	$(CLANG) -c -o $@ $(CFLAGS) $(word 2,$^) 
+
+.PHONY: clean
+
+clean:
+	$(RM) $(OUT_OBJ_DIR)/*.o
+	$(RM) -d $(OUT_OBJ_DIR)
+	$(RM) -d $(OUTDIR)/tmp/*
+	$(RM) -d $(OUTDIR)/tmp
+	$(RM) $(OUTDIR)/data/*
+	$(RM) -d $(OUTDIR)/data
+	$(RM) $(OUTDIR)/*.*
+	$(RM) $(OUTDIR)/*
```

_Can_ we say that an error can be ignored?

---
In reply to: [73583033](https://github.com/Microsoft/WinObjC/pull/816#discussion_r73583033) [](ancestors = 73583033)